src/node_crypto.cc

```diff
@@ -3952,6 +3954,37 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment* env, v8::Local<v8::Object> options,
```

Comment:
>> I guess I should move default values to JS as well, such that the native code does not need to handle undefined values.
>
> Yes, that's a good idea.

But what if `args[3]->IsInt32()` etc. returns false/fails? Should I just skip these tests, throw an exception, return silently...?